src/redux/logic/__tests__/reposDataHandlingLogic.spec.js

```diff
@@ -0,0 +1,168 @@
+import { fetchRepositoryLogic, fetchLanguagesLogic } from '../reposDataHandlingLogic'
+import * as types from '../../../actions/types'
+
+describe('fetchRepositoryLogic', () => {
+  describe('using existing repository name', () => {
```

Comment:
Add verifying url called in the process, done function eval, so cover test to 100% of testable file